src/DocumentDB.ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net451</TargetFrameworks>
```

Comment:
We'll probably find out during the Build and Release process, but I'm not 100% sure if the build agents have 4.5. net45's support ended quite a while ago.

Reply:
You need to update `<ItemGroup Condition="'$(TargetFramework)' == 'net451'">` to `<ItemGroup Condition="'$(TargetFramework)' == 'net45'">` down the line.

Reply:
good catch, thanks!

Reply:
Updated. Note that in both cases is is building just fine.